lib/contentful/request.ex

```diff
@@ -0,0 +1,131 @@
+defmodule Contentful.Request do
+  @moduledoc """
+  Encapsulates functions invloved in making a request towards the Contentful APIs.
+  """
+  alias Contentful.Configuration
+  alias Contentful.Query
+
+  import Contentful.Misc, only: [fallback: 2]
+
+  @agent_header [
+    "User-Agent": "Contentful Elixir SDK"
```

Comment:
User Agent headers should conform to our `X-Contentful-User-Agent` format, you can find the definition specs here: https://github.com/contentful/contentful.rb/blob/25027ea6cff613af15da27042b88584302c268cf/spec/client_configuration_spec.rb#L223

Reply:
Thanks! I think I copied that from an earlier stage of the code.

Reply:
Btw that would be an extra header. I still believe that a standard UA header can still exists, and since Contentful does not care about that one too much, we should be free to set it to a value we like.
Also as a non-official SDK the rules are not that strict anyway.

Reply:
Well, I better be prepared :wink: